### ISSUE TITLE
Not possible to import Retire.js Scan JSON results

### ISSUE THREAD

**Issue by john-trell** [DefectDojo/django-DefectDojo#4363]
**Bug description**
Uploading JSON for Retire.js Scan does not work.

It is because this line references a function that does not exists:

https://github.com/DefectDojo/django-DefectDojo/blob/master/dojo/tools/retirejs/parser.py#L19

**Steps to reproduce**
Post a JSON report for the Retire.js Scan type through the import-scan API.

**Expected behavior**
That you can upload a Retire.js Scan result.

**Additional context** (optional)
Incorrect line of code is here:
https://github.com/DefectDojo/django-DefectDojo/blob/master/dojo/tools/retirejs/parser.py#L19
It was removed as part of this commit and diff:
https://github.com/DefectDojo/django-DefectDojo/commit/6278a725bb5cfcac727fb318483bd178df3bec65#diff-6bf29b47605d741db6b4e3e5339565cc6560e7eb755d2ee8eb367d19c2f432d9L20

**Comment by damiencarol** [comment]
@john-trell taking a look.

**Comment by damiencarol** [comment]
I'm able reproduce it. Working on a pull request to fix that.
```
======================================================================
ERROR: test_parse (dojo.unittests.tools.test_retirejs_parser.TestRetireJsParser)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/damien/dd/dojo/unittests/tools/test_retirejs_parser.py", line 11, in test_parse
    findings = parser.get_findings(testfile, Test())
  File "/home/damien/dd/dojo/tools/retirejs/parser.py", line 19, in get_findings
    tree = self.parse_json(json_output)
AttributeError: 'RetireJsParser' object has no attribute 'parse_json'

----------------------------------------------------------------------
Ran 1 test in 0.004s

FAILED (errors=1)
```

**Comment by valentijnscholten** [comment]
fixed in dev / 1.15.0
